portal/src/util/projectname.test.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/* global describe, it, expect */
+import {
+  determineWord,
+  getRandom32BitsNumber,
+  maskNumber,
+  randomProjectName,
+} from "./projectname";
+
+// eslint-disable-next-line no-undef
+const mGetRandomValues = jest.fn().mockReturnValueOnce(new Uint32Array(1));
+Object.defineProperty(window, "crypto", {
+  value: { getRandomValues: mGetRandomValues },
+});
```

Comment:
What is the intention here? jsdom does not provide `window.crypto`, or you just want to mock `window.crypto`? If the case is the latter, refer to the documentation of Jest to mock globals.